crates/nu-command/tests/commands/mod.rs

```diff
@@ -8,6 +8,7 @@ mod cal;
 mod cd;
 mod compact;
 mod continue_;
+mod convertions;
```

Comment:
i just think this should be called "*conversions*", not "*convertions*", right? :open_mouth:

Reply:
thanks :blush: 

done in eb6d9f8